cdap-docs/tools/cdap-default/cdap-default-deprecated.xml

```diff
@@ -139,10 +139,7 @@
     <name>metadata.updates.publish.enabled</name>
     <value>false</value>
     <description>
-      Determines if metadata updates will be published to Apache Kafka.
-      External systems can subscribe to the Kafka topic determined by
-      ${metadata.updates.kafka.topic} to receive notifications of metadata
-      updates (deprecated).
+      Determines if metadata updates will be published to Apache Kafka (deprecated)
```

Comment:
The problem is that I was going to cherry-pick this to 3.5.1 & 3.5.2. Doing this stuff here first isn't working very well, is it?